GVFS/GVFS.Common/Http/HttpRequestor.cs

```diff
@@ -31,22 +42,63 @@ static HttpRequestor()
             availableConnections = new SemaphoreSlim(ServicePointManager.DefaultConnectionLimit);
         }
 
-        public HttpRequestor(ITracer tracer, RetryConfig retryConfig, GitAuthentication authentication)
+        protected HttpRequestor(ITracer tracer, RetryConfig retryConfig, Enlistment enlistment)
         {
-            this.client = new HttpClient(new HttpClientHandler() { UseDefaultCredentials = true });
-            this.client.Timeout = retryConfig.Timeout;
             this.RetryConfig = retryConfig;
-            this.authentication = authentication;
+
+            this.authentication = enlistment.Authentication;
 
             this.Tracer = tracer;
 
+            var httpClientHandler = new HttpClientHandler() { UseDefaultCredentials = true };

```

Comment:
Please remove all uses of `var` since we don't use that in this code base.  Thanks!

Reply:
Sure, went and rooted out every one of them in the files, that I've touched.